README.md

```diff
@@ -218,19 +218,45 @@ When running OpenShift in a single VM, it is recommended to reuse the Docker dae
 for pure Docker use-cases as well.
 By using the same docker daemon as Minishift, you can speed up your local experiments.
 
-To be able to work with the docker daemon on your Mac or GNU/Linux host use the
-[`docker-env command`](https://minishift.io/docs/minishift_docker-env.html) in your shell:
+1. Make sure that you have `docker` and `docker-client` installed on your machine.
```

Comment:
-1 to this change in this form. We are targeting all three operating systems - Linux, OS X and Windows. This documentation is Linux specific and in fact even specific to a certain distribution (Fedora I believe). Package names are different on various distributions and also how you control services. 

For Windows and OS X this instructions don't apply and in fact these are the two main platforms where one most likely want to use Minishift's Docker daemon. On Linux is makes handly sense imo, since there I rather run Docker natively. Different story on OS X and Windows where I need a VM to run Docker. In this case, it makes sense to "save" running an additional instance and use Minishift's Docker daemon instead. 

I think it is just not feasible to document how one has to install Docker. There are two many variations here. If anything we can add a note, that one needs the docker client.